### PR TITLE
VOTE-221 Disable page description fields on content types

### PR DESCRIPTION
**Pull request by rayestrada**
## JIRA Ticket
https://cm-jira.usa.gov/browse/VOTE-221

## Description

Page description fields were introduced in anticipation of teaser content or metadata descriptions. Teasers are not required for launch and the metadata descriptions can already be manipulated on each node. Until further requirement becomes necessary we are disabling.

### Post-deploy

1. `lando retune`

### QA/Test

1. Visit http://vote-gov.lndo.site/node/add and view the add form for each content type
2. Make sure there is no page description field displaying in the add form

## Checklist for the Developer

- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.
- [x] The result of these changes meet the JIRA ticket "definition of done".
- [x] This work meets the project [standards](/usagov/vote-gov-drupal/blob/dev/docs/standards.md).

## Checklist for the Peer Reviewers

- [x] The code has been reviewed.
- [ ] The file changes are relevant to the task.
- [x] The author of the commits match the assignee.
- [x] Code standards and best practices are followed.
- [x] QA/Test steps have been successfully completed and the results match "definition of done".
- [x] Drupal database log and browser console log are free of errors.
- [x] There are no known side-effects outside the expected behavior.
- [x] Code is readable and includes appropriate commenting.

### CODE REVIEW

File: config/sync/language/fr/metatag.metatag_defaults.global.yml
Comment (SamiraMSadat) [inline config/sync/language/fr/metatag.metatag_defaults.global.yml:1]: Is this change related to this ticket?

Comment (rayestrada) [reply config/sync/language/fr/metatag.metatag_defaults.global.yml:1]: I've reverted the change. Thanks.

**Review by SamiraMSadat — APPROVED** [review]
